README.md

```diff
@@ -26,15 +26,23 @@ We also provide reusable GO packages.
 
 ## Contents
 
-- [Applications](#applications)
-
+- [Contents](#contents)
+- [Examples](#examples)
+  - [Application](#application)
+  - [Packages](#packages)
 - [Contributing](#contributing)
 - [Contributors](#contributors)
 
-## Applications
+## Examples
+
+### Application
```

Comment:
The README file is autogenerated. You need to change this [file](https://github.com/nanlabs/nancy.go/blob/main/README.md.tmpl) instead without updating the TOC manually